azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeTelecomManagerIntegrationMode.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+import com.azure.android.core.util.ExpandableStringEnum;
+
+import java.util.Collection;
+
+/**
+ * CallCompositeTelecomManagerIntegrationMode {@link CallCompositeTelecomManagerIntegrationMode}.
+ */
+public final class CallCompositeTelecomManagerIntegrationMode
```

Comment:
Expectations for more values here?

I've definitely argued always enum, but ARB may ask, why not a boolean if only two values. 

Also, could identify in this value the default mode in the documentation.

Reply:
Adam, I think we decided always to use Enum not boolean for extensibility. + It have two modes so Enum is perfect.